app/components/phase_banner/style.scss

```diff
@@ -0,0 +1,5 @@
+.app-phase-banner--no-border {
+  .govuk-phase-banner {
+    border-bottom: 0;
```

Comment:
I'm wondering if there's a govuk css utility class to apply this rather than define a separate stylesheet?

Reply:
I'm not 100% sure but this is the way that Apply have done it, and this is also how it's done in the prototype so i'm happy with this